contents/windows/introduction.tw2

```diff
@@ -1,8 +1,7 @@
 ::Windows Introduction
 # Installing Exercism on Windows
 
-Are you using Windows 10?
+As a user of a Windows operating system you have the choice of using an installer application to symplify the installtion process of the Exercism CLI or if you are a Windows 10 user you may opt to install the CLI manually.  The walk-through can show you the proper steps to take.
```

Comment:
s/symplify/simplify/
s/installtion/installation/

Reply:
I'm wondering if we can reframe this to encourage using the installer application. Something like:

> The installer application will figure out what version of the Exercism CLI you need, download it, and put it in the right place.
> If you are on Windows 10, you may opt to install the CLI manually by following the steps provided.
>
> -> Install the CLI using the installer application.
> -> Proceed with manual installation.